package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "pdf.js",
   "version": "2.0.0",
+  "main": "build/lib/pdf.js",
```

Comment:
This must be reverted, since that's not at all desired/correct.

Please remember that the `lib` folder mostly exists for running the unit-tests on Travis, and the officially supported way of using the library is through the built `pdf.js`/`pdf.worker.js` files.